policyengine_us/parameters/gov/states/de/tax/income/subtractions/elderly_disabled/amount.yaml

```diff
@@ -0,0 +1,24 @@
+description: Delaware provides this exclusion amount to filers based on filing status.
+JOINT:
+  2021-01-01: 4_000
+HEAD_OF_HOUSEHOLD:
```

Comment:
did we check that head of household is treated as single?

Reply:
we couldn't find any policy for head of household, but we treated it as single because they are both for individuals

Reply:
> did we check that head of household is treated as single?
we couldn't find any policy for head of household, but we treated it as single because they are both for individuals